File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/FeedbackEndpoint.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class FeedbackEndpoint{

	@SerializedName("uiActions")
	private UiActions uiActions;

	@SerializedName("feedbackToken")
	private String feedbackToken;

	public void setUiActions(UiActions uiActions){
		this.uiActions = uiActions;
	}

	public UiActions getUiActions(){
		return uiActions;
	}

	public void setFeedbackToken(String feedbackToken){
		this.feedbackToken = feedbackToken;
	}

	public String getFeedbackToken(){
		return feedbackToken;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/FeedbackEndpoint.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class FeedbackEndpoint : Parcelable, Serializable {
    @SerializedName("uiActions")
    var uiActions: UiActions? = null

    @SerializedName("feedbackToken")
    var feedbackToken: String? = null
    override fun toString(): String {
        return "FeedbackEndpoint{" +
                "uiActions = '" + uiActions + '\'' +
                ",feedbackToken = '" + feedbackToken + '\'' +
                "}"
    }
}